### DISCUSSION TITLE
404: NOT_FOUND

### DISCUSSION THREAD

**Discussion by THEONECH7** [vercel/community#3848] [Help] [answered]
### Summary

I keep getting the error page every time I deploy a new project on vercel
404: NOT_FOUND
Code: NOT-FOUND

I am merely trying to host my website

### Example

https://web-2-three.vercel.app/

### Steps to Reproduce

https://github.com/THEONECH7/WEB2.git

**Comment by amyegan** [comment, marked as answer]
Hey @THEONECH7. It looks like your site works as expected when I visit https://web-2-three.vercel.app/Project1.html

If you want that page to load at https://web-2-three.vercel.app/, please change the file name from `Project1.html` to `index.html`. That will tell the browser that this is the default page to load as the homepage.

**Comment by THEONECH7** [comment]
Oh wow! It worked just fine.

Thank you Amy!

**Comment by exekyelmurga** [comment]
It worked for me too! Thanks you!
